zerocopy-derive/tests/ui-msrv/enum.stderr

```diff
@@ -67,54 +67,39 @@ error: TryFromBytes not supported on enum types
    | |_^
 
 error: must have a non-align #[repr(...)] attribute in order to guarantee this type's memory layout
-<<<<<<< HEAD
-  --> tests/ui-msrv/enum.rs:42:21
+  --> tests/ui-msrv/enum.rs:42:35
    |
-42 | #[derive(FromZeros, FromBytes)]
-   |                     ^^^^^^^^^
-   |
-   = note: this error originates in the derive macro `FromBytes` (in Nightly builds, run with -Z macro-backtrace for more info)
-
-error: only C-like enums can implement FromZeros
-=======
-  --> tests/ui-msrv/enum.rs:42:36
-   |
-42 | #[derive(TryFromBytes, FromZeroes, FromBytes)]
-   |                                    ^^^^^^^^^
+42 | #[derive(TryFromBytes, FromZeros, FromBytes)]
+   |                                   ^^^^^^^^^
    |
    = note: this error originates in the derive macro `FromBytes` (in Nightly builds, run with -Z macro-backtrace for more info)
 
 error: TryFromBytes not supported on enum types
   --> tests/ui-msrv/enum.rs:70:1
    |
-70 | / enum FromZeroes1 {
+70 | / enum FromZeros1 {
 71 | |     A(u8),
 72 | | }
    | |_^
 
-error: only C-like enums can implement FromZeroes
->>>>>>> [derive] Support `derive(TryFromBytes)` for structs
+error: only C-like enums can implement FromZeros
   --> tests/ui-msrv/enum.rs:70:1
    |
 70 | / enum FromZeros1 {
 71 | |     A(u8),
 72 | | }
    | |_^
 
-<<<<<<< HEAD
-error: only C-like enums can implement FromZeros
-=======
```

Comment:
Uh oh. The fact that this slipped through suggests that these tests aren't actually getting validated...